.github/ISSUE_TEMPLATE/timezone.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Is this a new package?

Reply:
Yes, this moves package:timezone from https://github.com/srawlins/timezone here.